### DISCUSSION TITLE
lineChart mergeMode

### DISCUSSION THREAD

**Discussion by zinkabqlejy65** [patrykandpatrick/vico#277] [Ideas]
Can lineChart achieve the same mergeMode effect as columnChart? My x-axis labels need to be merged into days according to dates for display.

![image](https://github.com/patrykandpatrick/vico/assets/114723517/27a8493c-4d7d-40f0-9159-01ff567560ef)

**Comment by patrickmichalik** [comment]
Hello! I’m afraid that, as such, this isn’t possible at the moment, but we’re currently working on an update that will introduce such functionality. I’ll be moving this discussion to the “Ideas” category.

**Comment by patrickmichalik** [comment]
I wanted to follow up and let you know that the changes introduced in [Vico 1.7.0](https://github.com/patrykandpatrick/vico/releases/tag/v1.7.0), [1.7.1](https://github.com/patrykandpatrick/vico/releases/tag/v1.7.1), and [1.7.2](https://github.com/patrykandpatrick/vico/releases/tag/v1.7.2) should enable you to achieve the result you’re looking for. `getXStep` enables you to lock the _x_ step and add intermediate entries, while `AxisItemPlacer.Horizontal` gives you full control over how a `HorizontalAxis` lays out its labels, ticks, and guidelines.
